.snapcraft/resources/restoredb

```diff
@@ -1,72 +1,88 @@
-#! /bin/bash
+#!/bin/bash
 
-if [[ ${EUID} != 0 ]]
-then
+function warn {
+        echo "[!] ${1}"
+        echo "[*] Check ${RESTORE_DIR}/${LOG_NAME} for details."
+}
+
+function abort {
+        echo "[!] ${1}"
+        echo "[*] Check ${RESTORE_DIR}/${LOG_NAME} for details."
+        echo "[-] Restore aborted!"
+        exit
+}
+
+if [[ ${EUID} != 0 ]]; then
     echo "[-] This task must be run with 'sudo'."
     exit
 fi
 
-backup_file=${1}
-if [[ ! -f ${backup_file} ]]
-then
-    echo "[-] Usage: snap run rocketchat-server.restoredb ${SNAP_COMMON}/backup_file.tgz"
+echo "*** ATTENTION ***"
+echo "* Your current database WILL BE DROPPED prior to the restore!"
+echo "* Do you want to continue?"
+
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) break;;
+        No ) exit;;
+    esac
+done
+
+if $(ps x | grep "node ${SNAP}/main.js" | grep -qv "grep"); then
+    echo "[-] Please shutdown Rocket.Chat first to restore a clean backup"
+    echo "[-] Use 'sudo systemctl stop snap.rocketchat-server.rocketchat-server'"
+    echo "[-] Backup aborted!"
```

Comment:
This one for absolute sure has to change.  Very good change to include!  :+1:  Solves a good number of community support issues :smile:

Reply:
So.. nothing to address, right? :smile:

Reply:
You're good on this one :)